addons/events/src/components/Event.js

```diff
@@ -149,7 +149,7 @@ export default class Item extends Component {
         </button>
         <Textarea
           id={`addon-event-${name}`}
-          ref={ref => {
+          inputRef={ref => {
```

Comment:
Do we even need all those refs? I can't find any usages of them

Reply:
remove em, if there are no references 👍